src/main/java/ch/uzh/ifi/hase/soprafs24/entity/Task.java

```diff
@@ -114,4 +114,16 @@ public void setId(Long id) {
 
     public void setDuration(int duration) { this.duration = duration; }
 
```

Comment:
This was previously done with the applicationsRepository. However, since this is a simple many to many relationship, adding a candidate to a Task can be done directly within the Task entity.